docs/configuring-metabase/environment-variables.md

```diff
@@ -1689,6 +1689,15 @@ Default: `null`
 
 The port for [MB_DB_HOST](#mb_db_host).
 
+
+### `MB_DB_QUERY_TIMEOUT_MINUTES`
+
+Type: integer<br>
+Default: `180`
+
+Timeout in minutes for query execution, both Metabase application database and data connections. In case you're execute a query and run into a timeout, you might consider increasing this value.
```

Comment:
Is this really meant to be 3 hours? The `defsetting` below says 3 minutes.

Reply:
Those docs are auto generated as per https://github.com/metabase/metabase/pull/44759#discussion_r1680017204. I've checked output of metabase.cmd.env-var-dox/generate-dox! to verify that file has correct content.